scripts/run_dependencies

```diff
@@ -14,7 +14,7 @@ PORT=5000
 docker volume create $VOLUME
 docker run --rm -v $VOLUME:/beebop/storage \
        --pull always \
-       ghcr.io/bacpop/beebop-py:$API_IMAGE \
+       mrcide/beebop-py:$API_IMAGE \
```

Comment:
Not sure why these have reverted?

Reply:
something wonky with a merge.. fixed thanks for the catch